src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double area() {
+        return Math.PI * (radius * radius);
```

Comment:
The formula for the area of a circle should be `Math.PI * Math.pow(radius, 2)` instead of `Math.PI * (radius * radius)`. Although both are mathematically equivalent, using `Math.pow` is more expressive and aligns with common mathematical notation.